packages/flutter/lib/src/material/i18n/localizations.dart

```diff
@@ -221,7 +309,14 @@ const Map<String, Map<String, String>> localizations = const <String, Map<String
     "closeButtonTooltip": r"Schließen",
     "nextMonthTooltip": r"Nächster Monat",
     "previousMonthTooltip": r"Vorheriger Monat",
+    "nextPageTooltip": r"Nächste Seite",
+    "previousPageTooltip": r"Vorherige Seite",
+    "showMenuTooltip": r"Zeige das Menü",
     "licensesPageTitle": r"Lizenzen",
+    "pageRowsInfoTitle": r"$firstRow–$lastRow von $rowCount",
+    "pageRowsInfoTitleApproximate": r"$firstRow–$lastRow von etwa $rowCount",
+    "rowsPerPageTitle": r"Zeilen pro Seite:",
```

Comment:
If you want to add it: "selectedRowCountTitleZero" in German would be "Kein Artikel ausgewält"